lib/Notes/NoteViewPage/components/NoteView/tests/note-view-test.js

```diff
@@ -58,6 +58,44 @@ describe('NoteView', () => {
     it('should display default Referred Record element', () => {
       expect(noteView.defaultReferredRecord.isPresent).to.be.true;
     });
+
+    it('should display correct note type', () => {
+      expect(noteView.noteType).to.equal(noteData.type);
+    });
+
+    it('should display correct note title', () => {
+      expect(noteView.noteTitle).to.equal(noteData.title);
+    });
+  });
+
+  describe('rendering NoteView component without type and title', () => {
+    setupApplication();
+    const noteDataWithoutTypeAndTitle = {
+      content: undefined,
+      links: [{
+        id: '8460928a-0cc9-482f-b68a-1f0db7624439',
+        type: 'request',
+      }],
```

Comment:
we have already declared this property, so better to use it here
```suggestion
      links: noteData.links,
```

or use `lodash,omit`
```
import omit from 'lodash/omit'
...
const noteDataWithoutTypeAndTitle = omit(noteData, ['type', 'title']);
```

Reply:
Like second variant. Thanks

Reply:
It makes literally no difference here, but as an alternative to increasing your dependance on lodash you could also destructure an object with `...rest` instead of using `omit`.

```
const { type, title, ...noteDataWithoutTypeAndTitle } = noteData;
```

While we're learning approaches I figured I'd throw it out there! 😄